docs/tools/civibuild.md

```diff
@@ -262,7 +262,29 @@ There are four variations on rebuilding. In order of fastest (least thorough) to
 
 ## Settings {:#settings}
 
-### civicrm.settings.php {:#settings-civicrm}
+### civicrm.settings.d folders {:#settings-civicrm}
+
+Civibuild provides a mechanism to quickly add settings to *all sites* which you've built with civibuild.
+
+For example, you can create a file `/etc/civicrm.settings.d/300-debug.php` with the following content to enable debugging and backtraces for all civibuild sites (useful for local development).
+
+```php
+<?php
+$GLOBALS['civicrm_setting']['domain']['debug_enabled'] = 1;
+$GLOBALS['civicrm_setting']['domain']['backtrace'] = 1;
+```
+
+Any settings which you would typically put in your site's `civicrm.settings.php` file can go into a php file (you choose the file name) in a `civicrm.settings.d` folder. Further, there are multiple `civicrm.settings.d` folders. Settings files from these directories will be loaded in the following order:
+
+1. `$PRJDIR/app/civicrm.settings.d/`
+1. `$PRJDIR/app/config/$TYPE/civicrm.settings.d/`
+1. `/etc/civicrm.settings.d/`
+1. `$SITE_DIR/civicrm.settings.d/`
+
+If a settings file has the *same name* as one which has already been loaded, it will be skipped.
```

Comment:
I think that leads to a backwards interpretation. Also, it's easy to confuse order-of-execution (based on filename) and order-of-override (based on folder).

The dynamic might be a bit clearer if we label the folders by their intended use, ie

 1. `$PRJDIR/app/civicrm.settings.d/` (General defaults provided by upstream buildkit for all civibuild sites)
 1. `$PRJDIR/app/config/$TYPE/civicrm.settings.d/`  (General defaults provided by upstream buildkit for specific types of sites)
 1. `/etc/civicrm.settings.d/`  (Overrides provided by the sysadmin for the local server)
 1. `$SITE_DIR/civicrm.settings.d/` (Overrides provided for a specific site/build)

For concrete example, suppose we have these files:
 * `$PRJDIR/app/civicrm.settings.d/200-two.php`
 * `$PRJDIR/app/civicrm.settings.d/300-three.php`
 * `/etc/civicrm.settings.d/100-one.php`
 * `/etc/civicrm.settings.d/300-three.php`

Then we would execute/load in this order:
 * `100-one.php` (specifically `/etc/civicrm.settings.d/100-one.php`; this is the only version of `100-one.php`)
 * `200-two.php` (specifically `$PRJDIR/app/civicrm.settings.d/200-two.php`; this is the only version of `200-two.php`)
 * `300-three.php` (specifically `/etc/civicrm.settings.d/300-three.php`; the system configuration in `/etc` overrides the stock code in `$PRJDIR/app/civicrm.settings.d`)